### PR TITLE
New website

### PR DESCRIPTION
**Pull request by DKbyo**
* Add css from template
* Change header
* Change footer
* Add API to retrieve info
* Add Experience boilerplate
* Merge resumes in the same json

Bugs
* Server render doesn't work

### CODE REVIEW

**Comment by erichulburd** [comment]
``` js
/* global require */

import i18n from 'i18next';
import LanguagesApi from 'api/languages.api';

function customLoad(url, options, loadComplete, data) {
  let lng = url; // load path must be set to "{{lng}}"
  let languages = {
    en: require('bhf-assets/translations/en.translation')
  };
  console.log("Custom loading of translations ", url, options, loadComplete, data);
  // load the file for given language and namespace
  let lapi = new LanguagesApi();
  lapi.get_translation(languages[lng]).then((result) => {
    console.log("Translation Loaded normally", result);
    return loadComplete(result, {status: 200});
  }).catch((error) => {
      console.log("Translation Loaded with error", error);
    return loadComplete({},  {status: 404});
  });
}

function i18nFactory(environment, directory, provider, callback) {
  let BackendProvider = provider || require('i18next-xhr-backend');
  let path = (environment === 'server') ? `${directory}/../assets/translations/{{lng}}.translation` : "{{lng}}";
  let loader = (environment === 'server') ? undefined : customLoad;

  let options = {
    fallbackLng: 'en',
    lng: 'en',

    // have a common namespace used around the full app
    ns: ['translation'],
    defaultNS: 'translation',

    debug: false,
    /* nsSeparator: false,
    keySeparator: false, */

    interpolation: {
      escapeValue: false // not needed for react!!
    },


    backend: {
      // path where resources get loaded from
      loadPath: path,

      // your backend server supports multiloading
      // /locales/resources.json?lng=de+en&ns=ns1+ns2
      allowMultiLoading: false,
      // allow cross domain requests
      crossDomain: true,

      parse: (data) => data,

      ajax: customLoad
    }
  };

  return i18n
    .use(BackendProvider)
    .init(options, callback);

}


export default i18nFactory;


```

**Comment by erichulburd** [comment]
se usa: http://i18next.com/

File: server/config/production/server.js [hunk 1]
```diff
@@ -22,13 +21,16 @@ class Server extends serverRenderable(ServerBase) {
     var server = this;
 
     // returns a Compiler instance
-    webpack(webpack_config, function(err, stats) {
+    //webpack(webpack_config, function(err, stats) {
+    
       server.config();
+      server.app.use('/assets', express.static(path.resolve(__dirname, '../assets')));
```
Comment (manue1) [inline server/config/production/server.js]: Doesn't this already get set correctly in server.config() ?

File: server/config/production/server.js [hunk 2]
```diff
@@ -22,13 +21,16 @@ class Server extends serverRenderable(ServerBase) {
...
+      server.app.set('views', path.resolve(__dirname, 'views'));
 
       server.app.listen(APP_PORT, () => {
         console.info(`App is now running on ${os.hostname()}`);
```
Comment (manue1) [inline server/config/production/server.js]: We'll probably always have it running on localhost, so better
`console.info(`App is now running on http://localhost:${APP_PORT}`);`

File: shared/components/sections/team/team.rt.html [hunk 3]
```diff
@@ -11,7 +11,7 @@ <h3 class="sub-title"><a href="#">{member.name}</a></h3>
                     <div class="meta">{member.title}</div>
                     <div class="action"><a href="#">Find out more</a></div>
                 </div><!--//content-->
-                <a class="link-mask" onClick="{this.goToMember.bind(this, memberIndex)}"></a>
```
Comment (manue1) [inline shared/components/sections/team/team.rt.html]: Could you please also remove the method from the component? :grimacing:

**Review by manue1 — CHANGES_REQUESTED** [review]
You have missed adding `webpack-node-externals` as a dependency.

Also please move all production-relevant packages to dependencies rather than dev. I guess there's just `webpack-dev-server` that's dev-only.

File: server/config/production/server.js (hunk 1 above)
Comment (DKbyo) [reply server/config/production/server.js]: You're right, I forgot delete this, when I setup the production server correctly

File: server/config/production/server.js (hunk 2 above)
Comment (DKbyo) [reply server/config/production/server.js]: Done!

File: shared/components/sections/team/team.rt.html (hunk 3 above)
Comment (DKbyo) [reply shared/components/sections/team/team.rt.html]: Don1

**Pull request merged**